review/tests/test_review_manager.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from datetime import timedelta
+from django.utils import timezone
+from review.models import Review
+
+
+@pytest.fixture
+def reviews():
+    now = timezone.now()
```

Comment:
once migration data is changes that test will fail
Note, did you think of it?

Reply:
> once migration data is changes that test will fail Note, did you think of it?

Should I remove migration data?

Reply:
It depends,
Anyway be aware it could cause failures in your tests if migration dat is changed